backend/tracim_backend/lib/webdav/resources.py

```diff
@@ -593,7 +593,12 @@ def move_folder(self, destpath):
                     self.content_api.save(self.content)
                 # move file if needed
                 if destination_workspace != self.content.workspace or destination_parent != self.content.parent :
-                    self.content_api.move_recursively(self.content, destination_parent, destination_workspace)
+                    self.content_api.move(
+                        self.content,
+                        new_parent=destination_parent,
+                        new_workspace=destination_workspace,
+                        must_stay_in_same_workspace=False
```

Comment:
@inkhey are you sure you can always set `must_stay_in_same_workspace` to False ? If so, the parameter is useless. If it may be true, then here we must have 2 cases : changing parent folder, and changing workspace

Reply:
@lebouquetin I do think this parameter is now useless, but i didn't check more about it.
We also support changing workspace and folder at the same time. so cases are a bit more complex.